### ISSUE TITLE
Git blame and history don't seem to work for files inside subdirectories of UnityPackages

### ISSUE THREAD

**Issue by MegaJiXiang** [modesttree/Projeny#68]
Hi! This is Jody from Magnopus. Sadly I went through upgrading to Projeny only to find out the blame and history no longer work. Seemingly because code files are inside of a symbolically linked folder instead of a "real" folder, tools to check the history or annotations are failing. For instance, I can go to the git command prompt and navigate to one of these linked sub-directories in project/Assets/linkedDirectoryX and try to run git blame on one of the code files. It will simply say that it can't find that file in source control. The same happens if I try to see the history. Similar problems also happen in Visual Studio 2017 where I right-click inside a code file, go to Source Control at the bottom, and pick Annotate or History. In theory this is because Git will navigate up the parent directories to find .git but it doesn't work with the linked folders.

Do you have any solutions for this problem? Thanks for your time.

**Comment by svermeulen** [comment]
Yes, I've noticed this problem myself.  I found a work around in the text editor I use, so it doesn't bother me too much anymore.

The only way I can think of maybe addressing this would be to change projeny to use the actual paths to the source files rather than the ones inside the directory link.  That could work in theory.
